### DISCUSSION TITLE
Recommended jsconfig.json for JS projects?

### DISCUSSION THREAD

**Discussion by ghost** [vuejs/language-tools#442] [Q&A] [answered]
If a project doesn't plan to use TS, what would be recommended `jsconfig.json` to better work in Volar?

**Comment by johnsoncodehk** [comment, marked as answer]
The latest version should not need to work with jsconfig, you will get same behaviors as *.js, if you want to create a jsconfig, please refer to https://vuejs.org/v2/guide/typescript.html#Recommended-Configuration.

**Reply by ghost** [reply]
What about having `tsconfig.json` but using JS throughout a project. Does it make sense?

**Reply by johnsoncodehk** [reply]
`tsconfig.json` + `"allowJs": true` ~= `jsconfig.json`, just feel free to choose.
